api/v1alpha1/dynakube_types.go

```diff
@@ -181,6 +181,10 @@ type RoutingSpec struct {
 	CapabilityProperties `json:",inline"`
 }
 
+type MetricsIngestSpec struct {
+	CapabilityProperties `json:",inline"`
+}
+
```

Comment:
Was this re-added on purpose? Currently it is unused as we switched to a feature flag annotation.

Reply:
by mistake, fixed